.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Hi, thank you for your contribution to Mantl! Before we can accept any code into
+master, we need it to meet the following criteria. If there are any you can't
+satisfy yourself, go ahead and open the issue anyway and we'll help you test.
+Feel free to delete this message once you're done. Thanks again!
+
+- [ ] Installs cleanly on a fresh build of most recent master branch
+- [ ] Upgrades cleanly from the most recent release
+- [ ] Updates documentation relevant to the changes
+- [ ] Rebases cleanly onto the latest master
```

Comment:
Yeah, that's a good point. I'm certainly up for going in that direction but we need to make sure everyone involved is familiar with how to do that.

In this comment, thought, I was more getting at the fact that some PRs (albeit from core committers) will not always be against master with our new branching strategy.